src/wp-includes/media.php

```diff
@@ -2349,6 +2349,14 @@ function _wp_post_thumbnail_context_filter_remove() {
  * @return string HTML content to display the caption.
  */
 function img_caption_shortcode( $attr, $content = '' ) {
+	/**
```

Comment:
```suggestion
	/*
```
Remove extra star from comment

Reply:
That is not suggested by WPCS.

A regex search of the WordPress core codebase for `\/\*[^\*]*]\s*$` does not find any example of the suggestion already in use.

Reply:
Check Multiline comment formate: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments

Reply:
Fixed now.